app/src/main/java/dk/jens/backup/AppInfo.java

```diff
@@ -0,0 +1,89 @@
+package dk.jens.backup;
```

Comment:
The thing is, that I used as a base the code from that other android app, simply deleting irrelevant stuff and adding relevant stuff. In the copyright sense it's good to keep some reference to the origin of some of the code.